src/diffpy/utils/tools.py

```diff
@@ -131,3 +133,25 @@ def get_package_info(package_names, metadata=None):
         pkg_info.update({package: importlib.metadata.version(package)})
     metadata["package_info"] = pkg_info
     return metadata
+
+
+def compute_mu_using_xraydb(sample, energy, density=None):
+    """Compute mu using the XrayDB database.
+
+    Reference: https://xraypy.github.io/XrayDB/python.html#xraydb.material_mu
+
+    Parameters
+    ----------
+    sample str
+        the chemical formula or the name of the material
+    energy float
+        the energy in eV
+    density float or None
+        material density in gr/cm^3
```

Comment:
this should be "sample mass density" or even better, "mass density of the packed powder/sample" to be clear.  the material desnsity is something quite different (mass of atoms in the unit cell divided by unit cell volume)